app/_hub/kong-inc/acme/0.2.11.md

```diff
@@ -216,6 +224,19 @@ $ curl https://mydomain.com
 # Now gives you a valid Let's Encrypt certicate
 ```
 
+#### Renew certificates
+
+The plugin automatically renews all certificate that are due for renewal everyday. Note the
```

Comment:
I think we should mention what's the criteria for "due for renewal". e.g. how many days before expiration does renewal efforts begin and what would happen if renewals fail (retry behavior, etc).

Reply:
added